src/module/constElements.js

```diff
@@ -0,0 +1,18 @@
+const refreshTask = document.querySelector('#refresh');
+const taskInput = document.querySelector('#task');
+const submitInput = document.querySelector('#arrow-submit');
+const taskList = document.querySelector('#taskList');
+const clearCompletedTasks = document.querySelector('#clear-completed');
```

Comment:
- [ ] OPTIONAL
Select checkboxes are not reflected in the local storage. Kindly ensure that these changes are stored in the local storage

Reply:
Based on the project sequence, this feature will be enabled in the next project addition
Thank you for the review.

Reply:
@Estherstarr 
Well noted 😃 🙏